### ISSUE TITLE
Support for gutter errors, navigate-to-line

### ISSUE THREAD

**Issue by atomdmac** [AtomLinter/linter-swagger#41]
Upon installing the plug-in and opening up a Swagger file, the linter does appear to run and find issues with my code where I expect it to. The results are displayed in the linter panel at the bottom of the editor.  However:
 - There are no line numbers included under the "Line" column
 - There are no markers in the gutter to denote where an error has been found
 - Clicking a row in the Linter Errors panel doesn't link to the location of the error.

Basically, I am now aware of the fact that I have errors to fix but have no way to know where they are :P

Is this the expected behavior?  I've made sure to install the `swagger-parser` npm package but I'm still not having any luck.

Thanks for any help you can provide!

**Comment by Arcanemagus** [comment]
Hmmm, no, that isn't the expected behavior. For example if you open the `sample1.json` file in the spec\fixtures folder of this package you should see the following:
![image](https://cloud.githubusercontent.com/assets/427137/26078069/25fdbf8c-3973-11e7-8712-b8af3bd94038.png)

**Comment by atomdmac** [comment]
Thanks for your reply!  After some more digging, it looks like the issue was that my swagger file was in a folder that was named in my `.gitignore` file.  (The swagger file is for a shared repo and is for my personal use so I didn't want to commit it).  Saving the file elsewhere allowed the errors to show along with the appropriate gutter markings and line numbers.

This is obviously pretty easy to work around, particularly if you know why the file is being ignore, but the cause wasn't immediately obvious to me.  Was this intentional?

Thanks again!

**Comment by Arcanemagus** [comment]
Hmmm, by default [Linter](https://atom.io/packages/linter) ignores files that are in the VCS ignore list (`.gitignore` here). This is controlled by the Atom wide setting "Exclude VCS Ignored Paths".

The odd part about that though is that it shouldn't have given you _any_ messages for one of those files.

**Comment by atomdmac** [comment]
Ah, I understand.  The behavior I was seeing makes sense, then.  

As far as why I saw messages when I should not have, I'm not sure.  I'm not _currently_ seeing any messages for ignored files so it's very possibly user error.  I'll keep an eye out for similar issues in the future but for now, I think we can close this 

Cheers!
